extensions/vertx-http/runtime/src/main/java/io/quarkus/vertx/http/runtime/devmode/Hashes.java

```diff
@@ -0,0 +1,26 @@
+package io.quarkus.vertx.http.runtime.devmode;
+
+import java.nio.charset.StandardCharsets;
+import java.security.MessageDigest;
+import java.security.NoSuchAlgorithmException;
+
+public class Hashes {
```

Comment:
We already have a `io.quarkus.deployment.util.HashUtil` around. Wondering if it makes sense to have everything in the same place?

(Moved to core runtime if the deployment artifact is not present here)